app/src/main/java/com/github/libretube/ui/views/CustomExoPlayerView.kt

```diff
@@ -695,11 +695,19 @@ open class CustomExoPlayerView(
         if (PlayerHelper.swipeGestureEnabled && this::brightnessHelper.isInitialized) {
```

Comment:
Due to this line this will only be executed when swipe gestures are enabled.
Please move this condition around the `brightnessHelper.restoreSavedBrightness()` calls below to fix this.

Otherwise lgtm.

Reply:
Like this?
![Screenshot 2023-11-19 at 3 17 02 PM](https://github.com/libre-tube/LibreTube/assets/98202630/71550907-b92b-4410-9b10-31583eb8b57b)

Reply:
Yes, exactly. Please do the same in the `else` block where `brightnessHelper.resetToSystemBrightness()` is called, then we're good to merge this :+1: